daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load ("//daml-foundations/daml-ghc/package-database:util.bzl", "bundled_package_db", "daml_package_rule", "daml_package_db")
+
+bundled_package_db(
+    name = "package-db",
+    pkg_db = ":package_db_for_damlc",
+    daml_lf_versions = ["1.0", "1.1", "1.2", "1.3"],
+    visibility = ["//visibility:public"],
+)
+
+daml_package_rule(
```

Comment:
Can we make this a rule which is parameterised with the DAML-LF version?

Reply:
I think this rule creates just all targets. @moritzkiefer-da is cleaning up the package-db rules, can you include @martin-drhu-da suggestions in your PR?